hub/core/storage/memory.py

```diff
@@ -1,19 +1,86 @@
-from hub.core.storage.mapped_provider import MappedProvider
-import fsspec  # type: ignore
+from hub.core.storage.provider import StorageProvider
 
 
-class MemoryProvider(MappedProvider):
+class MemoryProvider(StorageProvider):
     """Provider class for using the memory."""
 
     def __init__(self, root):
-        """Initializes the MemoryProvider.
+        self.mapper = {}
```

Comment:
Didn't we scrap the mapper? What's this doing here?

Reply:
OH I see. It's just a data dict. In that case, can I have a docstring for this and can we rename mapper?